.github/workflows/slither.yml

```diff
@@ -11,13 +11,36 @@ on:
       - release/**
 
 jobs:
+  get-diff:
```

Comment:
we should combine even this workflow inside the ci.yml file, makes sense now to keep only one job for get-diff

Reply:
Done